config/initializers/assets.rb

```diff
@@ -9,8 +9,3 @@
 # Add additional assets to the asset load path
 # Rails.application.config.assets.paths << Emoji.images_path
 
-# Precompile additional assets (application.js.coffee.erb, application.css, and all non-JS/CSS are already added)
-Rails.application.config.assets.precompile += %w( diagram.js graphing.js map_marker.js ace.js )
```

Comment:
Yeah ... that is a good question I don't know the answer to. I suspected that it would not work with additional configuration, but somehow it does:
```
root@38d92fad1b9d:/app# ls -al public/assets/*.js
-rw-r--r-- 1 root root 803641 Feb 24 19:06 public/assets/ace-b7a9a36a7e16893758aa7910f8b51810fb1673b0a7f8b5495c369cfddf6751e7.js
-rw-r--r-- 1 root root 432321 Feb 24 21:08 public/assets/ace-edaac82bab5f8f31f17d02dac1f82c356dab951ea95e48d977a9b0022a5b263c.js
-rw-r--r-- 1 root root 301207 Feb 24 21:08 public/assets/application-af5d3ed2164b4c06a6ab10e891ff4e4cc24e3b8227e3c9b45b0190220f5c31ae.js
-rw-r--r-- 1 root root 759678 Feb 24 19:06 public/assets/application-b53519a18e5154acfef3f481b6eb549f433c297ce5b5d20ff4f9c0e80a3e9a13.js
-rw-r--r-- 1 root root    613 Feb 24 21:08 public/assets/diagram-778f361aa4ed8e87fe51c63d2c61bde329dbf89fc57f31a3f044ec3f2e9154b5.js
-rw-r--r-- 1 root root    987 Feb 24 19:06 public/assets/diagram-8c003dba8f5583de4723c10c9633cbca33de071949984908797f6d478cc0e819.js
-rw-r--r-- 1 root root 166668 Feb 24 21:08 public/assets/graphing-141c9aa5934e3446123bf8e6b735e9d731fa2c9c803ef997c5a73ad72ad090ea.js
-rw-r--r-- 1 root root 333654 Feb 24 19:06 public/assets/graphing-5aa056a62d2dc93d05c4bd2f3d9c63395659962a7b923debb11d4b0d942fb8f5.js
-rw-r--r-- 1 root root      0 Feb 24 21:08 public/assets/manifest-d0ff5974b6aa52cf562bea5921840c032a860a91a3512f7fe8f768f6bbe005f6.js
-rw-r--r-- 1 root root      3 Feb 24 19:06 public/assets/manifest-e761884522a9ca2dac475b68bac1946c990fb9af8b676d3b6b45692a920c8d04.js
-rw-r--r-- 1 root root   1350 Feb 24 19:06 public/assets/map_marker-06756fac4a853bdcde5557d1f2e0d5333441903cf97ec1e75414e5871cc1af22.js
-rw-r--r-- 1 root root    356 Feb 24 21:08 public/assets/map_marker-d4e91cde532b7818afd0e878e3acc4f49e84af9e006976b5abfb08b5a865c549.js
```
My guess is the rules defined in `manifest.js` mean that every file in `app/assets/javascript` gets precompile into a separate js bundle